opentracing-api/src/main/java/io/opentracing/propagation/Adapters.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2016-2018 The OpenTracing Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.opentracing.propagation;
+
+import java.nio.ByteBuffer;
+
+public final class Adapters {
```

Comment:
what is the benefit of putting helpers here instead of `BinaryAdapters.injectCarrier()`?

Reply:
I remember we had this idea of putting all the adapters in a single place (including the `TextMap` ones) - else, I can definitely make `BinaryAdapters` public, and remove the `Adapters` class.

Reply:
I don't mind a shared class, but I'm bothered by the name "injectBinary", it reads like a command. We should come up with a more intuitive name, eg injectionCarrier, which works better with BinaryAdapters class.